server/src/test/java/org/elasticsearch/indices/IndicesQueryCacheTests.java

```diff
@@ -345,6 +353,79 @@ public void testStatsOnEviction() throws IOException {
         cache.close(); // this triggers some assertions
     }
 
+    public void testSkipCachingFactorAtLeast10() throws Exception {
```

Comment:
Small comment -- to me this test doesn't add much except guard against someone accidentally changing the skip factor. I think it'd accomplish the same to add a code comment like "the skipFactor must be 10 here to match the default in Lucene's LRUQueryCache".